app/build.gradle.kts

```diff
@@ -1,196 +1,36 @@
 import com.google.gms.googleservices.GoogleServicesPlugin.GoogleServicesPluginConfig
 
 plugins {
-    alias(libs.plugins.android.application)
-    alias(libs.plugins.kotlin.android)
-    alias(libs.plugins.ksp)
-    alias(libs.plugins.kotlin.parcelize)
+    alias(libs.plugins.homeassistant.android.application)
     alias(libs.plugins.firebase.appdistribution)
     alias(libs.plugins.google.services)
-    alias(libs.plugins.hilt)
-    alias(libs.plugins.compose.compiler)
+    alias(libs.plugins.homeassistant.android.dependencies)
 }
 
 android {
-    namespace = "io.homeassistant.companion.android"
-
-    compileSdk = libs.versions.androidSdk.compile.get().toInt()
-
-    ndkVersion = "21.3.6528147"
```

Comment:
I don't see anything in the codebase that needs the NDK and decided to remove this to avoid future issue.

Reply:
Following that logic, I think you can also remove [`AndroidApplicationConventionPlugin` line 95-97](https://github.com/home-assistant/android/pull/5078/files#diff-6f219b940c62174bcc616df1cf6d1e1342046ae168ca268f5d858920d87465eaR95-R97).

Reply:
I do plan to bring back test pretty soon that's why I kept it.